Make acceptable delay in check_chronos_jobs configurable

oktopuz opened the pull request:
internal tickets: PAASTA-11186, PAASTA-11277

solarkennedy commented on paasta_tools/monitoring_tools.py:
```diff
@@ -45,7 +45,8 @@ def monitoring_defaults(key):
         'project': None,
-        'realert_every': -1
+        'realert_every': -1,
+        'chronos_job_acceptable_delay': 15 * 60,  # 15 minutes
```
I would rather this not be a first-class key on the monitoring dictionary, and instead by an input to a chronos config. (without a prefix of "chronos")

solarkennedy commented on paasta_tools/check_chronos_jobs.py:
```diff
@@ -224,7 +224,12 @@ def sensu_message_status_for_jobs(chronos_job_config, service, instance, cluster
         else:
-            acceptable_delay = 15 * 60
+            monitoring_overrides = chronos_job_config.get_monitoring()
+            acceptable_delay = monitoring_tools.get_chronos_job_acceptable_delay(
```
Is the use-case here that people want to give chronos more time before an alert, if so, why is `alert_after` not enough?

oktopuz commented:
* removed acceptable_delay
* increased alert_after for the chronos jobs check to 15m to compensate acceptable_delay removal
* refactored the check to make it more precise for crontab-like schedules with non-fixed interval (0 * * * 1-5, 0 0 */2 * * and etc)
